src/views/transaction-details/transaction-details.view.tsx

```diff
@@ -29,7 +29,7 @@ const TransactionDetails: FC = () => {
       <Header title="Transaction Details" />
       <Card className={classes.card}>
         <div className={classes.balance}>
-          <TokenIcon token={token} className={classes.tokenIcon} size={48} />
+          <TokenIcon logoURI="" className={classes.tokenIcon} size={48} />
```

Comment:
Why are all these passed empty?

![image](https://user-images.githubusercontent.com/15896177/163993218-a29c2b23-a1a4-4800-9006-c15da0de517f.png)

Reply:
Nope, wasn't conscious. Fixed now to always use ETH logo by the moment!